src/AzureSqlAction.ts

```diff
@@ -90,15 +88,50 @@ export default class AzureSqlAction {
 
     private async _executeSqlFile(inputs: ISqlActionInputs) {
         core.debug('Begin executing sql script');
-        let scriptContents: string;
-        try {
-            scriptContents = fs.readFileSync(inputs.sqlFile, "utf8");
+
+        // sqlcmd should be added to PATH already, we just need to see if need to add ".exe" for Windows
+        let sqlCmdPath: string;
+        switch (process.platform) {
+            case "win32": 
+                sqlCmdPath = "sqlcmd.exe";
+                break;
+            case "linux":
+            case "darwin":
+                sqlCmdPath = "sqlcmd";
+                break;
+            default:
+                throw new Error(`Platform ${process.platform} is not supported.`);
         }
-        catch (e) {
-            throw new Error(`Cannot read contents of file ${inputs.sqlFile} due to error '${e.message}'.`);
+
+        // Determine the correct sqlcmd arguments based on the auth type in connectionConfig
+        let sqlcmdCall = `"${sqlCmdPath}" -S ${inputs.serverName} -d ${inputs.connectionConfig.Config.database}`;
```

Comment:
@shueybubbles Please take a look at my go-sqlcmd calls to see if the arguments are correct. Couldn't add you as a PR reviewer, maybe because you aren't in the Azure org on github.